docs/portal/developer-portal/uan_admin_guide.ditamap

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE map PUBLIC "-//OASIS//DTD DITA Map//EN" "map.dtd">
 <map id="uan_admin">
-    <title>HPE Cray Supercomputing User Access Node (UAN) Software Administration Guide (S-8033)</title>
+    <title>HPE Cray Supercomputing User Access Node (UAN) Software Administration Guide (S-8033) 2.7.0</title>
     <topicmeta>
         <shortdesc>This publication provides instructions and examples for managing and troubleshooting User Access Nodes (UANs) on an HPE Cray EX supercomputer.</shortdesc>
         <data name="pubsnumber" value="S-8033"></data>
-		<data name="edition" value="UAN Software Release @product_version@"></data>
```

Comment:
Can we leave the `@product_version@` replacement in place and use it everywhere? It will save us having to continually update this.

Reply:
I tried that first, but instead of getting a simple SemVer string like `2.7.0` in the published docs, I keep getting uglier ones like `2.7.0-cdaf3654-dirty` inserted into the docs and their metadata. Is it because I build the docs locally?

Reply:
Yes, it should come out clean on the tagged build.